docs/docs/getting-started/building-an-image.md

```diff
@@ -0,0 +1,32 @@
+---
+sidebar_position: 3
+title: Build a Kernel
+---
+
+After making any required modifications to your kernel, the project uses
+```shell
+docker buildx bake
+```
+
+to build the various Docker images. Each kernel can be targeted specifically,
+or if needed, any of the configured groups of images can be built together
+
+## Supported Build Targets
+| Bake Target | Language | Version | Notes                 |
+|---|---|---|---|
+| `python_3_9` | Python | 3.9 | CPU |
+| `python_3_10` | Python | 3.10 | CPU |
+| `python_3_9_gpu` | Python | 3.9 | GPU |
+| `python_3_10_gpu` | Python | 3.10 | GPU |
+| `rlang_4_3_0` | R | 4.3.0 | CPU |
+
+## Building a Kernel
+
+TODO: Cleanup around Task copying of files where needed
+
+Once necessary files have been copied, `bake` can be executed. This will build
+the requested target, and any required parent images
+```
+task prep:${target}
```

Comment:
this doesn't look like it exists yet:

```shell
task: Task "prep:python_3_10" does not exist

```

Reply:
I think @shouples was working on some of these, but I'll double back through them to see which commands are needed

Reply:
Taskfile invocations have been adjusted as needed